test/features/forms/MainDonation_UpgradeToYearly_CustomAmount_AddressType.ts

```diff
@@ -0,0 +1,56 @@
+import { VueWrapper } from '@vue/test-utils';
+import {
+	expectMainDonationFormGoesToPageOnSubmit,
+	submitMainDonationForm
+} from '@test/features/forms/subForms/MainDonationForm';
+import {
+	expectUpgradeToYearlyFormGoesToPageOnLinkClick,
+	expectUpgradeToYearlyFormGoesToPageOnSubmit
+} from '@test/features/forms/subForms/UpgradeToYearlyForm';
+import { Intervals } from '@src/utils/FormItemsBuilder/fields/Intervals';
+import { PaymentMethods } from '@src/utils/FormItemsBuilder/fields/PaymentMethods';
+import { expectCustomAmountFormGoesToPageOnSubmit } from '@test/features/forms/subForms/CustomAmountForm';
+import { expectAddressTypeFormSubmits } from '@test/features/forms/subForms/AddressTypeForm';
+import { AddressTypes } from '@src/utils/FormItemsBuilder/fields/AddressTypes';
+
+export const donationFormFeatures: Record<string, ( wrapper: VueWrapper<any> ) => Promise<any>> = {
+	expectMainDonationFormGoesToAddressFormWhenSofortIsSelected: ( wrapper: VueWrapper<any> ) => expectMainDonationFormGoesToPageOnSubmit(
+		wrapper,
+		4,
```

Comment:
I find these "magic numbers" confusing - I had to look at the function definition of `expectMainDonationFormGoesToPageOnSubmit` to see that the number is a page number, I have to guess that they start at 1, etc. I suggest to create constants in the file.